### PR TITLE
Add ICEPAY

### PR DESCRIPTION
**Pull request by viceromag**

### CODE REVIEW

**Review by XaosSintez — APPROVED** [review]

**Review by subenson — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

File: openapi/components/schemas/GatewayAccountConfig/ICEPAY.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: ICEPAY credentials object
+        properties:
+          merchantId:
+            type: string
+            description: ICEPAY merchant ID
+            format: password
```
Comment (slavcodev) [inline openapi/components/schemas/GatewayAccountConfig/ICEPAY.yaml:15]: Is it really sensitive to be `password`?

Comment (viceromag) [reply openapi/components/schemas/GatewayAccountConfig/ICEPAY.yaml:15]: I believe yes - it can be used for other API operations.

Comment (slavcodev) [reply openapi/components/schemas/GatewayAccountConfig/ICEPAY.yaml:15]: W/o secret?

Comment (slavcodev) [reply openapi/components/schemas/GatewayAccountConfig/ICEPAY.yaml:15]: I think it's like username, you need password to complete authentication.

**Pull request merged**